### ISSUE TITLE
Incorrect reporting of color format when combined with browser hacks

### ISSUE THREAD

**Issue by bartveneman** [projectwallace/css-analyzer#285]
From Gazelle.nl:

```css
color: #000\9;
```

is reported as `hex5`, but it should be `hex3`.
